app/views/tests/_form.html.erb

```diff
@@ -29,14 +29,14 @@
 
   <div class="field">
     <%= form.label :test_type %>
-    <%= form.select :test_type, options_for_select([['Approved Includes', 'approved_includes'], 
-                                                    ['Compile', 'compile'], 
-                                                    ['Coverage', 'coverage'], 
-                                                    ['I/O', 'i/o'], 
-                                                    ['Memory Errors', 'memory_errors'], 
-                                                    ['Performance', 'performance'], 
-                                                    ['Script', 'script'],
-                                                    ['Unit', 'unit']], @test.test_type) %>
+    <%= form.select :test_type, options_for_select([['approved_includes', 'approved_includes'], 
```

Comment:
See earlier comment. Also, what is this partial? It looks like it's duplicating what's already in our other _test_form partial.  Is it actually rendered anywhere?